config/webpack.config.dev.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
         loader: 'json'
       },
       {
-        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)$/,
+        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
relevant for `url(font.woff?v=1.3)` in some CSS libs. webpack 2 doesn't need this anymore.